shared/tracker/action.render.desktop.js

```diff
@@ -65,7 +65,10 @@ export default class ActionRender extends Component {
   renderNormal (styles: Object, username: string) {
     return (
       <div style={{...styles.container}}>
-        <Button waiting={this.props.waiting} style={styles.actionButton} type='Follow' label='Track' onClick={() => this.props.onFollow(username)} />
```

Comment:
So some of these tracker components kinda assume the tracker is shown only cause you don't track or something broke, but we can actually spawn these more passively (like with keybase id --ui) so really 'normal' can be used if you're following or not. we might want to refactor some of that kind of stuff more later